### PR TITLE
Specify distro name & version

### PR DESCRIPTION
**Pull request by jakirkham**
Allow Linux distro name to be specified with `DISTRO_NAME` ( needed for PR https://github.com/conda-forge/docker-images/pull/214 ). Also rename `CENTOS_VER` to `DISTRO_VER`.

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge/core, any thoughts on this? 🙂

**Review by bdice — APPROVED** [review]
All looks good to me, with one question about `$JAVA_DISTRO_VERSION` / `$JAVA_CENTOS_VERSION`.

File: scripts/setup_openjdk [hunk 1]
```diff
@@ -19,7 +19,7 @@ fi
 
 yum update -y
 yum install -y \
-  java-1.8.0-openjdk-headless-"$JAVA_CENTOS_VERSION" \
+  java-1.8.0-openjdk-headless-"$JAVA_DISTRO_VERSION" \
```
[the remark this answers is not in the document]
Comment (jakirkham) [reply scripts/setup_openjdk:22]: It's leftover from Jenkins images we had here. We should drop them. I'll just revert this change for now

[the remark this answers is not in the document]
Comment (jakirkham) [reply scripts/setup_openjdk:22]:
```suggestion
  java-1.8.0-openjdk-headless-"$JAVA_CENTOS_VERSION" \
```

[the remark this answers is not in the document]
Comment (jakirkham) [reply scripts/setup_openjdk]: Submitted PR ( https://github.com/conda-forge/docker-images/pull/216 ) to drop these scripts

**Comment by bdice** [comment]
@jakirkham Looks like a couple jobs had random HTTP failures. Can you rerun?
```
("Connection broken: ConnectionResetError(104, 'Connection reset by peer')", ConnectionResetError(104, 'Connection reset by peer'))
```

**Comment by jakirkham** [comment]
Yeah restarted

**Comment by jakirkham** [comment]
Now green 🟢

File: linux-anvil-aarch64/Dockerfile [hunk 2]
```diff
@@ -16,8 +19,8 @@ ENV LANGUAGE=en_US.UTF-8
 ADD https://loripsum.net/api /opt/docker/etc/gibberish
 
 # Resolves a nasty NOKEY warning that appears when using yum.
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER} && \
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER}-aarch64
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${DISTRO_VER} && \
+    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${DISTRO_VER}-aarch64
```
Comment (hmaarrfk) [inline linux-anvil-aarch64/Dockerfile:23]: aren't these two line still hyper specific to centos?

**Review by hmaarrfk — APPROVED** [review]

File: linux-anvil-aarch64/Dockerfile (hunk 2 above)
Comment (hmaarrfk) [reply linux-anvil-aarch64/Dockerfile:23]: understood thank you!

Comment (jakirkham) [reply linux-anvil-aarch64/Dockerfile:23]: Also have a follow up PR where we might be able to merge the Dockerfiles together conditioning on `DISTRO_NAME`, etc.. So should make these simpler to manage and update

**Comment by jakirkham** [comment]
Thanks Mark! 🙏

**Pull request merged**